### PR TITLE
Fix setup instructions

### PR DESCRIPTION
**Pull request by julienrf**
Consistent with https://github.com/scala/docs.scala-lang/pull/2698

### CODE REVIEW

File: _install_tabs/1-macos.html [hunk 1]
```diff
@@ -9,7 +9,10 @@
     {% assign homebrewAlt = "Alternatively, if you don't use Homebrew:" %}
```
Comment (bishabosha) [inline _install_tabs/1-macos.html:9]:
```suggestion
    {% assign homebrewAlt = "Alternatively for M1 Macs, or if you don't use Homebrew:" %}
```

**Review by bishabosha — APPROVED** [review]
generally I approve, but the home-brew link installs the x86 version so I think the alt-details label text should reflect this

**Comment by julienrf** [comment]
Ah, good point. But I wonder if we could fix the homebrew formula instead to download the correct binary.

**Comment by bishabosha** [comment]
> Ah, good point. But I wonder if we could fix the homebrew formula instead to download the correct binary.

then at that point, revert the title :)

**Pull request merged**